Gemfile

```diff
@@ -9,5 +9,5 @@ gem 'bump'
 gem 'pry'
 gem 'rake'
 gem 'rspec'
-gem 'rubocop'
+gem 'rubocop', '~> 1.50.2'
```

Comment:
This line downgrades the rubocop version, and it's probably not a good thing for newer versions. Maybe we can create an old version with these changes and don't push them to the main?

Reply:
Otherwise we can Specify a variable on the gemfile like:

if RUBY_VERSION > 2.6
  gem 'rubocop'
else
  gem 'rubocop', '~> 1.50.2'
end